### PR TITLE
update to vapor 2

### PR DESCRIPTION
**Pull request by LoganWright**

### CODE REVIEW

File: Sources/VaporForms/Provider.swift [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+    do {
+        let stem = try drop.stem()
+        let tags: [Tag] = [
+            ErrorsForField(),
+            IfFieldHasErrors(),
+            LabelForField(),
+            LoopErrorsForField(),
+            ValueForField()
+        ]
+        tags.forEach(stem.register)
```
Comment (BrettRToomey) [inline Sources/VaporForms/Provider.swift]: 💯

**Comment by bygri** [comment]
Well, thank you

**Comment by vzsg** [comment]
@bygri could you please tag a version (e.g. 0.6.0) with this?

**Comment by bygri** [comment]
As of yesterday, one of the beta dependencies wouldn't compile. I will get the chance to check again in a few hours.

**Comment by bygri** [comment]
Build is [still failing](https://travis-ci.org/bygri/vapor-forms/builds/219495954). I was going to wait for Vapor 2 to become more stable and then dig into this again.

**Comment by vzsg** [comment]
Ah, okay. I was just curious as the topic of using this lib with v2 came up on Slack.
Thanks!

**Pull request merged**